config.js

```diff
@@ -19,6 +19,10 @@
     "featureManagerIsSSLEnabled": false,
     "featureManagerIsWAFEnabled": true,
     "homePageCards": ["ApplyDefaultSettingsCard", "SSLCard", "PurgeCacheCard"],
+    "moreSettingsCards": {
+        "container.moresettings.speed": ["CacheLevelCard", "AlwaysOnlineCard", "ImageOptimizationCard"],
+        "container.moresettings.security": ["SecurityLevelCard"]
+    },
```

Comment:
The `cloudflare\cloudflare-frontend` repository needs to have every card in the category.  The plugin specific configs will have more specific lists.  After this PR is merged add CacheLevelCard, AlwaysOnlineCard, and ImageOptimizationCard to wordpress config, as well as SecurityLevelCard.